docs/bugs_and_glitches.md

```diff
@@ -1671,6 +1672,56 @@ This bug is why the Lapras in [maps/UnionCaveB2F.asm](https://github.com/pret/po
 +	jr nz, .noclip_tiles
 ```
 
+### Pokémon deposited in the Day Care might lose experience
+
+This bug happens because when retrieving a Pokémon only its level is updated, but its Exp. Points are set to the minimum required for that level. This means that, if a Pokémon is deposited, hasn't gained any level and then it's taken back, it might lose Exp. Points.
```

Comment:
```suggestion
This happens because when a Pokémon is withdrawn from the Day-Care, its Exp. Points are reset to the minimum required for its level. This means that if it hasn't gained any levels, it may lose experience.
```